src/components/ConfirmDialog.react.js

```diff
@@ -11,30 +12,54 @@ export default class ConfirmDialog extends Component {
 
     constructor(props) {
         super(props);
+        this.state = {
+            displayed: props.displayed
+        };
+        this._setStateAndProps = this._setStateAndProps.bind(this);
```

Comment:
I don't see _setStateAndProps being passed as a callback anywhere so this .bind() isn't necessary?